### ISSUE TITLE
Tabs With  Different height. Extra space in the shorter tab

### ISSUE THREAD

**Issue by sagrawalcb** [JungHsuan/react-native-collapsible-tabview#9]

**Comment by JungHsuan** [comment]
Yes. You can set ``minHeight`` to fit the shorter tab, but when you scroll the longer tab, the shorter one would not scroll to the right position because the ``scrollToOverflowEnabled`` is not implemented on Android. 
However, it would work on iOS.

**Comment by sagrawalcb** [comment]
Tried but not working.

**Comment by sagrawalcb** [comment]
So i want to reset all scroll positions to zero on the tab change event. Can you suggest me what to do about it?

**Comment by shuru-sagarj** [comment]
Instead of resetting, you could add dummy invisible components to the list (enough to make the list scrollable).
